editors/vscode/syntaxes/jakt.tmLanguage.json

```diff
@@ -522,7 +522,7 @@
           "include": "#this"
         },
         {
-          "match": "\\b(mut)\\s+(this)\\b",
+          "match": "\\b(mut)\\b",
```

Comment:
Capture 2 of this now points to the void, please remove that too (just below in captures.2)

Reply:
Indeed, fixed